docs/integrations/destinations/local-csv.md

```diff
@@ -49,7 +49,9 @@ The local mount is mounted by Docker onto `LOCAL_ROOT`. This means the `/local`
 If your Airbyte instance is running on the same computer that you are navigating with, you can open your browser and enter [file:///tmp/airbyte\_local](file:///tmp/airbyte_local) to look at the replicated data locally.
 If the first approach fails or if your Airbyte instance is running on a remote server, follow the following steps to access the replicated files:
 1. Access the scheduler container using `docker exec -it airbyte-scheduler bash`
-2. Navigate to the default local mount using `cd /tmp/airbyte_local`
-3. Navigate to the replicated file directory you specified when you created the destination, using `cd /{Directory_Specified}`
-4. List files containing the replicated data using `ls`
-5. Execute `cat {filename}` to display the data in a particular file
+1. Navigate to the default local mount using `cd /tmp/airbyte_local`
```

Comment:
List formatting got messed up here.

Reply:
I am bit puzzled too about this. The reason I made this change is based on the suggestion [here ](https://guides.github.com/features/mastering-markdown/) for ordered list. I thought of this change because on the main Airbyte Gitbook docs, the section gets rendered like a paragraph instead of order list. 
![image](https://user-images.githubusercontent.com/20258314/121997661-55b4d600-cdc8-11eb-9ea8-28a0fa0e9bf4.png)
With my change, where every line starts with "1.", the Github still renders properly, as below. Hoping Gitbook may also render properly once this change?
![image](https://user-images.githubusercontent.com/20258314/121997762-7c730c80-cdc8-11eb-9ad7-8ad9dd816b5e.png)